static/templates/settings/user-list-admin.handlebars

```diff
@@ -1,15 +1,23 @@
 <div id="admin-user-list" class="settings-section" data-name="user-list-admin">
-<div class="settings-section-title"><i class="icon-vector-user settings-section-icon"></i>
-{{t "Users" }}</div>
-<table class="table table-condensed table-striped wrapped-table">
-  <tbody id="admin_users_table" class="admin_user_table">
-    <th class="wrapped-cell">{{t "Name" }}</th>
-    <th>{{t "Email" }}</th>
-    <th class="last_active">{{t "Last active" }}</th>
-    {{#if is_admin}}
-        <th>{{t "Actions" }}</th>
-    {{/if}}
-  </tbody>
-</table>
-<div id="admin_page_users_loading_indicator"></div>
+    <div class="settings-section-title">
+        <div class="table-title">
+            <i class="icon-vector-user settings-section-icon"></i>
+            {{t "Users" }}
+        </div>
+        <input type="text" class="search" placeholder="{{t 'Search users' }}" />
+        <div class="clear-float"></div>
+    </div>
+    <div class="progressive-table-wrapper">
+        <table class="table table-condensed table-striped wrapped-table">
+            <thead>
+                <th class="wrapped-cell">{{t "Name" }}</th>
+                <th>{{t "Email" }}</th>
+                {{#if is_admin}}
+                    <th>{{t "Actions" }}</th>
+                {{/if}}
+            </thead>
+            <tbody id="admin_users_table" class="admin_user_table"></tbody>
+        </table>
+    </div>
+    <div id="admin_page_users_loading_indicator"></div>
```

Comment:
I think you lost the `last_active` column heading:

![image](https://cloud.githubusercontent.com/assets/2746074/25301560/3f59d854-26df-11e7-8de0-365ae71803d4.png)